docs/mpcnc/trucks.md

```diff
@@ -0,0 +1,127 @@
+# Assembling The Trucks
+
+Click on the images to embiggen.
+
+## Bearings
+![!MPCNC Truck parts](https://www.v1engineering.com/wp-content/uploads/2020/06/TParts-scaled.jpg){: width="400"}
+
+* You will need 5 sets of 5/16 x 1 1/2" (M8 x 40mm) Bolts and locking nuts.
+* Two 1/2" sockets and ratchets (or 13mm for M8's).
+* 5 Bearings.
+* Printed Truck and Truck Mirrored parts.
+
+___
+
+## Tension Bolts
+![!MPCNC truck tension bolts](https://www.v1engineering.com/wp-content/uploads/2020/06/Tension-scaled.jpg){: width="400"}
+
+* Pay attention to the direction of the bolt heads.
+* Insert a bearing in the truck and fit the bolts.
+* Just seat the locking nut, these will get adjusted later. Leave these loose.
+
```

Comment:
I don't have the hardware with me, but is this 2 bolts/truck? Maybe that's worth saying. I'm not sure how far to go in this step before I stop.